doc/modules/model_evaluation.rst

```diff
@@ -1756,6 +1759,148 @@ the same does a lower Brier score loss always mean better calibration"
     and probability estimation." <https://drops.dagstuhl.de/opus/volltexte/2008/1382/>`_
     Dagstuhl Seminar Proceedings. Schloss Dagstuhl-Leibniz-Zentrum fr Informatik (2008).
 
+.. _class_likelihood_ratios:
+
+Class likelihood ratios
+-----------------------
+
+The :func:`class_likelihood_ratios` function computes the `positive and negative
+likelihood ratios
+<https://en.wikipedia.org/wiki/Likelihood_ratios_in_diagnostic_testing>`_
+:math:`LR_\pm` for binary classes, which can be interpreted as the ratio of
+post-test to pre-test odds as explained below. As a consequence, this metric is
+invariant w.r.t. the class prevalence (the number of samples in the positive
+class divided by the total number of samples) and can be extrapolated between
+populations regardless of any possible class imbalance.
+
+The :math:`LR_\pm` metrics are therefore very useful in settings where the data
+available to learn and evaluate a classifier is a study population with nearly
+balanced classes, such as a case-control study, while the target application,
+i.e. the general population, has very low prevalence.
+
+The positive likelihood ratio :math:`LR_+` is the probability of a classifier to
+correctly predict that a sample belongs to the positive class divided by the
+probability of predicting the positive class for a sample belonging to the
+negative class:
+
+.. math::
+
+   LR_+ = \frac{\text{PR}(P+|T+)}{\text{PR}(P+|T-)}.
+
+The notation here refers to predicted (:math:`P`) or true (:math:`T`) label and
+the sign :math:`+` and :math:`-` refer to the positive and negative class,
+respectively, e.g. :math:`P+` stands for "predicted positive".
+
+Analogously, the negative likelihood ratio :math:`LR_-` is the probability of a
+sample of the positive class being classified as belonging to the negative class
+divided by the probability of a sample of the negative class being correctly
+classified:
+
+.. math::
+
+   LR_- = \frac{\text{PR}(P-|T+)}{\text{PR}(P-|T-)}.
+
+Notice that probabilities differ from counts, for instance
+:math:`\operatorname{PR}(P+|T+)` is not equal to the number of true positive
+counts ``tp``. In fact, the class likelihood ratios can be expressed in terms of
+the `sensitivity and specificity
+<https://en.wikipedia.org/wiki/Sensitivity_and_specificity>`_, which are further
+metrics derived from the :func:`confusion_matrix`:
+
+.. math::
+
+   \text{sensitivity} = \frac{tp}{tp + fn},
+
+.. math::
+
+   \text{specificity} = \frac{tn}{tn + fp},
+
+.. math::
+
+   LR_+ = \frac{\text{sensitivity}}{1 - \text{specificity}} = \frac{tp \times
+   (tn + fp)}{fp \times (tp + fn)},
+
+.. math::
+
+   LR_- = \frac{1 - \text{sensitivity}}{\text{specificity}} = \frac{fn \times
+   (tn + fp)}{tn \times (tp + fn)}.
```

Comment:
```suggestion
counts ``tp`` (see `the wikipedia page <https://en.wikipedia.org/wiki/Likelihood_ratios_in_diagnostic_testing>`_ for the actual formulas).
```

We should focus on the information important to our users (who are users, and not developers of machine-learning code) and avoid duplicating wikipedia because 1) it decreases maintenance (docs are like code: the more we have, the more maintenance), 2) it "crowds" the docs: people have a limited amount of time, and need to find information to make their decisions fast, if we put more information, it takes longer to find the relevant one.